plugins/apache-airflow/src/components/DagTableComponent/DagTableComponent.tsx

```diff
@@ -136,8 +186,26 @@ type DagTableComponentProps = {
 
 export const DagTableComponent = (props: DagTableComponentProps) => {
   const { dagIds } = props;
+  const [dagsData, setDagsData] = useState<DagTableRow[] | []>([]);
```

Comment:
```suggestion
  const [dagsData, setDagsData] = useState<DagTableRow[]>([]);
```

you don't need that part